packages/footer-component/src/components/dc-footer/dc-social-media.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+  tag: 'dc-social-media',
+  styleUrl: 'dc-footer.css',
+})
+export class DcLogo {
+  render() {
+    return (
+      <ul class="footer__social_media">
+        <li>
+          <a aria-label="Twitter link" href="https://twitter.com/0debtzone" target="_blank" rel="noreferrer noopener">
+            <svg aria-hidden="true" width="27" height="22" viewBox="0 0 27 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M26.3044 2.9264C25.3424 3.35816 24.307 3.6494 23.2206 3.77973C24.3298 3.1093 25.1803 2.04733 25.5814 0.780469C24.5439 1.4021 23.3954 1.85317 22.1719 2.09667C21.1928 1.04328 19.7973 0.384644 18.2529 0.384644C15.2881 0.384644 12.8842 2.81162 12.8842 5.80552C12.8842 6.22978 12.9315 6.6433 13.0234 7.04074C8.56099 6.8144 4.6043 4.65666 1.95607 1.37689C1.49389 2.17712 1.22933 3.10822 1.22933 4.10262C1.22933 5.98306 2.17706 7.64252 3.61779 8.61438C2.73805 8.58596 1.90985 8.34192 1.18577 7.9359C1.18524 7.95843 1.18524 7.98149 1.18524 8.00456C1.18524 10.6305 3.03608 12.821 5.49254 13.3198C5.04205 13.4431 4.56765 13.5096 4.07784 13.5096C3.73147 13.5096 3.3952 13.4759 3.06742 13.412C3.75113 15.5655 5.73372 17.1332 8.08287 17.1767C6.24531 18.6307 3.93069 19.4974 1.41473 19.4974C0.981772 19.4974 0.554123 19.4717 0.133911 19.4213C2.51068 20.9601 5.33263 21.8568 8.36443 21.8568C18.2407 21.8568 23.6408 13.5971 23.6408 6.43359C23.6408 6.19867 23.636 5.96428 23.6259 5.73151C24.6746 4.96882 25.5851 4.01358 26.3044 2.9264Z"
+                fill="white"
+              />
+            </svg>
+          </a>
+        </li>
+        <li>
+          <a aria-label="Facebook link" href="https://www.facebook.com/DebtCollective" target="_blank" rel="noreferrer noopener">
```

Comment:
prefer built-in WARIA attributes whenever is possible, here I think we can use `title`. Other examples will be `<div role="banner">` instead `<nav>` or `<img aria-label` instead `<img alt=""`... It is a wild guess but I think it is right...